### ISSUE TITLE
Theano Compilation Error when constructing random variable

### ISSUE THREAD

**Issue by TernaryM01** [pymc-devs/pymc#4474]
## Description of my problem

I just installed PyMC3, and tried to run the "Bayesian Methods for Hacker" [Jupyter notebook](https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/blob/master/Chapter1_Introduction/Ch1_Introduction_PyMC3.ipynb). When executing the following code:
```python
import pymc3 as pm
import theano.tensor as tt

with pm.Model() as model:
    alpha = 1.0/count_data.mean()  # Recall count_data is the
                                   # variable that holds our txt counts
    lambda_1 = pm.Exponential("lambda_1", alpha)
    lambda_2 = pm.Exponential("lambda_2", alpha)
    
    tau = pm.DiscreteUniform("tau", lower=0, upper=n_count_data - 1)
```

The following error occured:
```python
---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
<ipython-input-9-7eab6c64b9ee> in <module>
      8     alpha = 1.0/count_data.mean()  # Recall count_data is the
      9                                    # variable that holds our txt counts
---> 10     lambda_1 = pm.Exponential("lambda_1", alpha)
     11     lambda_2 = pm.Exponential("lambda_2", alpha)
     12 

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\pymc3\distributions\distribution.py in __new__(cls, name, *args, **kwargs)
    119             dist = cls.dist(*args, **kwargs, shape=shape)
    120         else:
--> 121             dist = cls.dist(*args, **kwargs)
    122         return model.Var(name, dist, data, total_size, dims=dims)
    123 

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\pymc3\distributions\distribution.py in dist(cls, *args, **kwargs)
    128     def dist(cls, *args, **kwargs):
    129         dist = object.__new__(cls)
--> 130         dist.__init__(*args, **kwargs)
    131         return dist
    132 

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\pymc3\distributions\continuous.py in __init__(self, lam, *args, **kwargs)
   1485         super().__init__(*args, **kwargs)
   1486         self.lam = lam = tt.as_tensor_variable(floatX(lam))
-> 1487         self.mean = 1.0 / self.lam
   1488         self.median = self.mean * tt.log(2)
   1489         self.mode = tt.zeros_like(self.lam)

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\tensor\var.py in __rtruediv__(self, other)
    174 
    175     def __rtruediv__(self, other):
--> 176         return theano.tensor.basic.true_div(other, self)
    177 
    178     def __rfloordiv__(self, other):

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\graph\op.py in __call__(self, *inputs, **kwargs)
    251 
    252         if config.compute_test_value != "off":
--> 253             compute_test_value(node)
    254 
    255         if self.default_output is not None:

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\graph\op.py in compute_test_value(node)
    124 
    125     # Create a thunk that performs the computation
--> 126     thunk = node.op.make_thunk(node, storage_map, compute_map, no_recycling=[])
    127     thunk.inputs = [storage_map[v] for v in node.inputs]
    128     thunk.outputs = [storage_map[v] for v in node.outputs]

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\graph\op.py in make_thunk(self, node, storage_map, compute_map, no_recycling, impl)
    632             )
    633             try:
--> 634                 return self.make_c_thunk(node, storage_map, compute_map, no_recycling)
    635             except (NotImplementedError, MethodNotDefined):
    636                 # We requested the c code, so don't catch the error.

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\graph\op.py in make_c_thunk(self, node, storage_map, compute_map, no_recycling)
    599                 raise NotImplementedError("float16")
    600         outputs = cl.make_thunk(
--> 601             input_storage=node_input_storage, output_storage=node_output_storage
    602         )
    603         thunk, node_input_filters, node_output_filters = outputs

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\basic.py in make_thunk(self, input_storage, output_storage, storage_map)
   1202         init_tasks, tasks = self.get_init_tasks()
   1203         cthunk, module, in_storage, out_storage, error_storage = self.__compile__(
-> 1204             input_storage, output_storage, storage_map
   1205         )
   1206 

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\basic.py in __compile__(self, input_storage, output_storage, storage_map)
   1140             input_storage,
   1141             output_storage,
-> 1142             storage_map,
   1143         )
   1144         return (

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\basic.py in cthunk_factory(self, error_storage, in_storage, out_storage, storage_map)
   1632             for node in self.node_order:
   1633                 node.op.prepare_node(node, storage_map, None, "c")
-> 1634             module = get_module_cache().module_from_key(key=key, lnk=self)
   1635 
   1636         vars = self.inputs + self.outputs + self.orphans

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\cmodule.py in module_from_key(self, key, lnk)
   1189             try:
   1190                 location = dlimport_workdir(self.dirname)
-> 1191                 module = lnk.compile_cmodule(location)
   1192                 name = module.__file__
   1193                 assert name.startswith(location)

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\basic.py in compile_cmodule(self, location)
   1548                     lib_dirs=self.lib_dirs(),
   1549                     libs=libs,
-> 1550                     preargs=preargs,
   1551                 )
   1552             except Exception as e:

C:\ProgramData\Anaconda3\envs\primer\lib\site-packages\theano\link\c\cmodule.py in compile_str(module_name, src_code, location, include_dirs, lib_dirs, libs, preargs, py_module, hide_symbols)
   2545             compile_stderr = compile_stderr.replace("\n", ". ")
   2546             raise Exception(
-> 2547                 f"Compilation failed (return status={status}): {compile_stderr}"
   2548             )
   2549         elif config.cmodule__compilation_warning and compile_stderr:

Exception: ('Compilation failed (return status=1): In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\include/Python.h:63,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:1:. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/pyport.h:699:2: error: #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?).".   699 | #error "LONG_BIT definition appears wrong for platform (bad gcc/glibc config?).".       |  ^~~~~.
In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\include/Python.h:156,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:1:. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:85:5: error: \'__int64\' does not name a type; did you mean \'__int64_t\'?.    85 |     __int64 st_size;.       |     ^~~~~~~.       |     __int64_t.
In file included from /usr/include/sys/stat.h:22,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\include/pyport.h:224,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\include/Python.h:63,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:1:. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:86:12: error: expected \';\' at end of member declaration.    86 |     time_t st_atime;.       |            ^~~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:86:12: error: expected unqualified-id before \'.\' token.    86 |     time_t st_atime;.       |            ^~~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:88:12: error: expected \';\' at end of member declaration.    88 |     time_t st_mtime;.       |            ^~~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:88:12: error: expected unqualified-id before \'.\' token.    88 |     time_t st_mtime;.       |            ^~~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:90:12: error: expected \';\' at end of member declaration.    90 |     time_t st_ctime;.       |            ^~~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/fileutils.h:90:12: error: expected unqualified-id before \'.\' token.    90 |     time_t st_ctime;.       |            ^~~~~~~~.
In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\include/Python.h:8,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:1:. C:\\ProgramData\\Anaconda3\\envs\\primer\\include/pyconfig.h:249:29: error: \'__int64\' does not name a type; did you mean \'__int64_t\'?.   249 | #       define PY_LONG_LONG __int64.       |                             ^~~~~~~. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:316:9: note: in expansion of macro \'PY_LONG_LONG\'.   316 | typedef PY_LONG_LONG npy_longlong;.       |         ^~~~~~~~~~~~.
In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarraytypes.h:4,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarrayobject.h:12,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayobject.h:4,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:5:. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:317:31: error: expected initializer before \'npy_ulonglong\'.   317 | typedef unsigned PY_LONG_LONG npy_ulonglong;.       |                               ^~~~~~~~~~~~~.
In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarraytypes.h:4,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarrayobject.h:12,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayobject.h:4,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:5:. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:618:17: error: \'npy_longlong\' does not name a type; did you mean \'npy_long\'?.   618 |         typedef npy_longlong npy_int64;.       |                 ^~~~~~~~~~~~.       |                 npy_long. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:619:17: error: \'npy_ulonglong\' does not name a type; did you mean \'npy_ulong\'?.   619 |         typedef npy_ulonglong npy_uint64;.       |                 ^~~~~~~~~~~~~.       |                 npy_ulong. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:1097:9: error: \'npy_int64\' does not name a type; did you mean \'npy_int8\'?.  1097 | typedef npy_int64 npy_timedelta;.       |         ^~~~~~~~~.       |         npy_int8. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/npy_common.h:1098:9: error: \'npy_int64\' does not name a type; did you mean \'npy_int8\'?.  1098 | typedef npy_int64 npy_datetime;.       |         ^~~~~~~~~.       |         npy_int8.
In file included from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarrayobject.h:12,.
from C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayobject.h:4,.
from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:5:. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarraytypes.h:769:9: error: \'npy_int64\' does not name a type; did you mean \'npy_int8\'?.   769 |         npy_int64 year;.       |         ^~~~~~~~~.       |         npy_int8. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/ndarraytypes.h:775:9: error: \'npy_int64\' does not name a type; did you mean \'npy_int8\'?.   775 |         npy_int64 day;.       |         ^~~~~~~~~.       |         npy_int8.
In file included from C:\\Users\\Mauri Sombowadile\\AppData\\Local\\Theano\\compiledir_Windows-10-10.0.19041-SP0-Intel64_Family_6_Model_158_Stepping_10_GenuineIntel-3.7.7-64\\tmp751gswav\\mod.cpp:6:. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayscalars.h:38:9: error: \'npy_longlong\' does not name a type; did you mean \'npy_long\'?.    38 |         npy_longlong obval;.       |         ^~~~~~~~~~~~.       |         npy_long. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayscalars.h:68:9: error: \'npy_ulonglong\' does not name a type; did you mean \'npy_ulong\'?.    68 |         npy_ulonglong obval;.       |         ^~~~~~~~~~~~~.       |         npy_ulong. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayscalars.h:121:9: error: \'npy_datetime\' does not name a type; did you mean \'npy_datetimestruct\'?.   121 |         npy_datetime obval;.       |         ^~~~~~~~~~~~.       |         npy_datetimestruct. C:\\ProgramData\\Anaconda3\\envs\\primer\\lib\\site-packages\\numpy\\core\\include/numpy/arrayscalars.h:127:9: error: \'npy_timedelta\' does not name a type; did you mean \'npy_timedeltastruct\'?.   127 |         npy_timedelta obval;.       |         ^~~~~~~~~~~~~.       |         npy_timedeltastruct. cc1plus: warning: unrecognized command line option \'-Wno-c++11-narrowing\'. ', 'FunctionGraph(Elemwise{true_div,no_inplace}(TensorConstant{1.0}, TensorConstant{0.05065023956194388}))')
```

My hardware:
CPU: Intel Core i7-9750H (9th gen)
GPU: NVIDIA RTX 2060 Mobile
(CUDA 10.1 and CuDNN 7.6 installed and tested successfully with Tensorflow and PyTorch)

## Versions and main components

* PyMC3 Version: 3.11.1
* Theano Version: Theano-PyMC 1.1.2
* Python Version: 3.7.7
* Operating system: Windows 10 Home 64-bit, version 20H2
* How did I install PyMC3: in conda environment, using pip install

**Comment by brandonwillard** [comment]
It looks like you might have the old Theano package installed.  This version of PyMC3 uses Theano-PyMC.

**Comment by TernaryM01** [comment]
No. I simply installed PyMC3 through pip install. The Theano package is Theano-PyMC included with installing PyMC3. I never installed Theano before.

**Comment by brandonwillard** [comment]
I see; there are a bunch of compilation errors to the far right in the unformatted output.  A lot are related to missing NumPy types and conflicts with other C-level dependencies.  

If you installed Theano-PyMC with `pip`, as well, you could try `pip uninstall`ing it and reinstalling with `conda`.

**Comment by TernaryM01** [comment]
I just tried, but the conda install (from conda-forge) failed. The error message printed on the conda prompt is:

```
C:\WINDOWS\system32>SET DISTUTILS_USE_SDK=1

C:\WINDOWS\system32>SET MSSdk=1

C:\WINDOWS\system32>SET "VS_VERSION=15.0"

C:\WINDOWS\system32>SET "VS_MAJOR=15"

C:\WINDOWS\system32>SET "VS_YEAR=2017"

C:\WINDOWS\system32>set "MSYS2_ARG_CONV_EXCL=/AI;/AL;/OUT;/out"

C:\WINDOWS\system32>set "MSYS2_ENV_CONV_EXCL=CL"

C:\WINDOWS\system32>set "PY_VCRUNTIME_REDIST=\bin\vcruntime140.dll"

C:\WINDOWS\system32>set "CXX=cl.exe"

C:\WINDOWS\system32>set "CC=cl.exe"

C:\WINDOWS\system32>set "VSINSTALLDIR="

C:\WINDOWS\system32>for /F "usebackq tokens=*" %i in (`vswhere.exe -nologo -products * -version [15.0,16.0) -property installationPath`) do (set "VSINSTALLDIR=%i\" )

C:\WINDOWS\system32>if not exist "" (for /F "usebackq tokens=*" %i in (`vswhere.exe -nologo -products * -requires Microsoft.VisualStudio.Component.VC.v141.x86.x64 -property installationPath`) do (set "VSINSTALLDIR=%i\" ) )

C:\WINDOWS\system32>if not exist "" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\" )

C:\WINDOWS\system32>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\" )

C:\WINDOWS\system32>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\" )

C:\WINDOWS\system32>if not exist "C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\" (set "VSINSTALLDIR=C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\" )

C:\WINDOWS\system32>IF NOT "" == "" (
set "INCLUDE=;"
 set "LIB=;"
 set "CMAKE_PREFIX_PATH=;"
)

C:\WINDOWS\system32>call :GetWin10SdkDir

C:\WINDOWS\system32>call :GetWin10SdkDirHelper HKLM\SOFTWARE\Wow6432Node  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE\Wow6432Node  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKLM\SOFTWARE  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 exit /B 1

C:\WINDOWS\system32>exit /B 0

C:\WINDOWS\system32>for /F %i in ('dir /ON /B "C:\Program Files (x86)\Windows Kits\10\\include\10.*"') DO (SET WindowsSDKVer=%~i )

C:\WINDOWS\system32>(SET WindowsSDKVer=10.0.17763.0 )

C:\WINDOWS\system32>(SET WindowsSDKVer=10.0.18362.0 )

C:\WINDOWS\system32>if errorlevel 1 (echo "Didn't find any windows 10 SDK. I'm not sure if things will work, but let's try..." )  else (echo Windows SDK version found as: "10.0.18362.0" )
Windows SDK version found as: "10.0.18362.0"

C:\WINDOWS\system32>IF "win-64" == "win-64" (
set "CMAKE_GEN=Visual Studio 15 2017 Win64"
 set "BITS=64"
)  else (
set "CMAKE_GEN=Visual Studio 15 2017"
 set "BITS=32"
)

C:\WINDOWS\system32>pushd C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\
The system cannot find the path specified.

C:\WINDOWS\system32>CALL "VC\Auxiliary\Build\vcvars64.bat" -vcvars_ver=14.16 10.0.18362.0
The system cannot find the path specified.

C:\WINDOWS\system32>popd

C:\WINDOWS\system32>IF "" == "" SET "CMAKE_GENERATOR=Visual Studio 15 2017 Win64"

C:\WINDOWS\system32>call :GetWin10SdkDirHelper HKLM\SOFTWARE\Wow6432Node  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE\Wow6432Node  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKLM\SOFTWARE  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 call :GetWin10SdkDirHelper HKCU\SOFTWARE  1>nul 2>&1

C:\WINDOWS\system32>if errorlevel 1 exit /B 1

C:\WINDOWS\system32>exit /B 0
```

So it seems to be looking for a Visual Studio 2017 installation, and failed, which is weird because vs2017 is in the list of packages to be installed:
```
The following NEW packages will be INSTALLED:

  filelock           conda-forge/noarch::filelock-3.0.12-pyh9f0ad1d_0
  libgpuarray        conda-forge/win-64::libgpuarray-0.7.6-hfa6e2cd_1003
  mako               conda-forge/noarch::mako-1.1.4-pyh44b312d_0
  pygpu              conda-forge/win-64::pygpu-0.7.6-py37hda49f71_1002
  python_abi         conda-forge/win-64::python_abi-3.7-1_cp37m
  scipy              pkgs/main/win-64::scipy-1.6.0-py37h14eb087_0
  theano-pymc        conda-forge/win-64::theano-pymc-1.1.2-py37hf2a7229_0
  vs2017_win-64      conda-forge/win-64::vs2017_win-64-19.16.27038-h2e3bad8_2
  vswhere            conda-forge/win-64::vswhere-2.8.4-h57928b3_0
```

I've seen this error message somewhere when somebody tried to install Tensorflow on Windows. It seems to be an issue with conda install; that's why I prefer pip install.

**Comment by TernaryM01** [comment]
Update: I just switched to Python 3.8.5 and installed PyMC3 again.

I got the exact same error message...

**Comment by TernaryM01** [comment]
I just realized that theano-pymc actually did get installed from conda-forge, despite the error message.

So I ran the notebook again, and still got the same problem...

**Comment by brandonwillard** [comment]
Perhaps your local Conda env, or a build script, is missing some environment variables and that's making it look for dependencies in system locations.  I don't run Windows, but it seems odd that a Conda-installed VS2017 would reside in your `C:\Program Files (x86)\` directory.  

In general, all of these problems could easily stem from the use of insufficient system-level dependencies instead of the ones installed in your Conda environment.  

You could try changing your system libraries to match Theano-PyMC's requirements, but that could be bad for other reasons.

**Comment by TernaryM01** [comment]
No, conda doesn't install VS2017 in Program Files. It simply installs a lightweight VS (runtime or something) in its own environment, but then gets confused.

Conda installs are very buggy on Windows, and conda uninstalls very likely lead to disasters. I once uninstalled a package in one conda environment, which ended up uninstalling it from all conda environments and also uninstalled some random other packages which broke all the links between packages. I ended up uninstalling Anaconda and cleared all of its cached packages.

However, I believe my issue with PyMC3 has nothing to do with the conda install problem. I've successfully installed many packages (almost all of which through pip install) and run Tensorflow and PyTorch with GPU utilization. My conda setup is fine.

In any case, I've given up on this issue, and installed PyMC3 in a conda environment in WSL 2. Finally, it works like a charm (although then I won't be able to utilize the GPU).
